masterlist.yaml

```
@@ -13180,7 +13187,27 @@ plugins:
     req: [ 'Better descriptions SSE.esp' ]
   - name: 'Better Descriptions SSE - YOT Patch.esp'
     req: [ 'YOT - Your Own Thoughts.esp' ]
-
+  - name: 'RealisticWaterTwo - Unique Region Names.esp'
+    msg:
+      - <<: *alreadyInX
+        subs: [ 'RealisticWaterTwo - Unique Region Names - Cutting Room Floor.esp' ]
+        condition: 'active("RealisticWaterTwo - Unique Region Names - Cutting Room Floor.esp")'
+      - <<: *alreadyInX
+        subs: [ 'RealisticWaterTwo - Unique Region Names - MissingEZs_AllExteriors.esp' ]
+        condition: 'active("RealisticWaterTwo - Unique Region Names - MissingEZs_AllExteriors.esp")'
+  - name: 'Unique Region Names.esp'
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/56452/' ]
+    group: *cellSettingsGroup
+    msg:
+      - <<: *patchProvided
+        subs: [ 'Realistic Water Two SE' ]
+        condition: 'active("RealisticWaterTwo.esp") and not active("Cutting Room Floor.esp") and not active("MissingEZs_AllExteriors.esp") and not active("RealisticWaterTwo - Unique Region Names.esp") and not active("RealisticWaterTwo - Unique Region Names - (Cutting Room Floor|MissingEZs_AllExteriors)\.esp")'
+      - <<: *patchProvided
+        subs: [ 'Realistic Water Two SE and Cutting Room Floor SSE' ]
+        condition: 'active("RealisticWaterTwo.esp") and active("Cutting Room Floor.esp") and not active("RealisticWaterTwo - Unique Region Names - Cutting Room Floor.esp")'
+      - <<: *patchProvided
+        subs: [ 'Realistic Water Two SE and Missing Encounter Zones FIXED' ]
+        condition: 'active("RealisticWaterTwo.esp") and active("MissingEZs_AllExteriors.esp") and not active("RealisticWaterTwo - Unique Region Names - MissingEZs_AllExteriors.esp")'
```

Add newlines between files from different mods as well.

```suggestion
        condition: 'active("RealisticWaterTwo.esp") and active("MissingEZs_AllExteriors.esp") and not active("RealisticWaterTwo - Unique Region Names - MissingEZs_AllExteriors.esp")'

```